lightcurve/static/lightcurve/js/lightcurve.js

```diff
@@ -74,18 +73,20 @@ $(function() {
     return false;
   });
 
-  $(document).on('click', '.edit-name', function(e) {
-    e.preventDefault();
+  $('.js-edit-name').on('click', function() {
     var filename = $(this).prev(".lightcurve-name").text();
     $( $(this).prev(".lightcurve-name") ).replaceWith( "<input type='name' class='input_name' name='input_name' value='" + filename + "'>" );
-    $(this).replaceWith("<button type='submit' class='btn btn-sm btn-primary submit-name'>Save</button><button class='btn btn-sm btn-default cancel'>Cancel</button>");
+    $(this).css('display', 'none');
```

Comment:
Shorthand for `.css('display', 'none')` is `.hide()`  (applicable here and elsewhere)

Reply:
Done.